inventory/group_vars/fi.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+# See https://github.com/openfoodfoundation/ofn-install/wiki/Setup for more info
+checkout_zone: Europe
+country_code: fi
+currency: EUR
+locale: fi                  # Default locale for OFN instance
+available_locales: fi,en    # Available locales for OFN instance
+language: fi_FI.UTF-8       # Default language for the server
+language_packages:          # Language packages for the server
+  - language-pack-en-base
+l10n_repo: https://github.com/jarihu/l10n_fi.git
+timezone: Helsinki
+
+
+# Used for the LetsEncrypt email setting
+developer_email: hello@openfoodnetwork.fi
+
+# Users listed here will have their key from /files/keys added to the app_user
+users_sysadmin:
+  - jari-fi
```

Comment:
Note that all other instances on in this repo are configured to allow the core dev team access to the server, in order to provide support, and deploy weekly releases.
```yaml
      - "{{ core_devs }}"
```

But we can also only provide this where an agreement has been made, including a funding commitment. I'm not sure if you've seen this yet, but the agreement is described here: https://docs.google.com/spreadsheets/d/1uHc3Pn-XSi5vJ95YIvWRtRJCbz3nXL5apdBWipmHUYc/edit?gid=2012212782#gid=2012212782

Of course, this is your choice, and you can choose later. I can see that you are quite capable of supporting the server yourself already.